cases/management/commands/enrich_ciaa_sources.py

```
@@ -0,0 +1,536 @@
+                f"Failed to create AG charge sheet source for case {case.case_id}"
+            )
+            self.stats["errors"] += 1
+            return False
+
+    def _enrich_press_releases(self, case: Case, dry_run: bool) -> bool:
+        enriched = False
+
+        defendants = self._get_defendant_names(case)
+        existing_sources = self._get_existing_sources_for_case(case)
+
+        for source_id, source in existing_sources.items():
```

Ruff (0.15.12)

[warning] Loop control variable `source_id` not used within loop body

Rename unused `source_id` to `_source_id`

(B007)